repose-aggregator/components/filters/valkyrie-authorization-filter/src/test/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilterTest.scala

```diff
@@ -596,26 +618,32 @@ class ValkyrieAuthorizationFilterTest extends FunSpec with BeforeAndAfterEach wi
     val transformedTenant = "98765"
     val contactId = "123456"
     val filterChain = mock[FilterChain]
-    val mockServletRequest = new MockHttpServletRequest
-    val mockServletResponse = new MockHttpServletResponse
+    var mockServletRequest = new MockHttpServletRequest
```

Comment:
Got it. The `MockHttpServletRequest` and `MockHttpServletResponse` `val`'s are defined at the individual test level within some test groups and the `var`'s are defined at the group level when there was common setup/initialization of it.